cfme/tests/infrastructure/test_system_image_type.py

```diff
@@ -60,3 +60,36 @@ def test_name_required_error_validation(appliance):
 #        provision_type='Vm')
 #    sys_image_type.create()
 #    sys_image_type.delete(cancel=False)
+
+
+def test_system_image_type_selective_delete(appliance):
+    """
+    Tests System Image Type for delete operation using select option on All page.
+    """
+    collection = appliance.collections.system_image_types
+    sys_image_type1 = collection.create(
+        name=fauxfactory.gen_alphanumeric(8),
+        provision_type=SystemImageType.VM_OR_INSTANCE)
+    sys_image_type2 = collection.create(
+        name=fauxfactory.gen_alphanumeric(8),
+        provision_type=SystemImageType.VM_OR_INSTANCE)
+    collection.delete(False, sys_image_type1, sys_image_type2)
+
+
+def test_system_image_type_delete_validation(appliance):
+    """
+    Tests System Image Type for correct flash message after delete operation.
+    """
+    collection = appliance.collections.system_image_types
+    sys_image_type1 = collection.create(
+        name=fauxfactory.gen_alphanumeric(8),
+        provision_type=SystemImageType.VM_OR_INSTANCE)
+    sys_image_type2 = collection.create(
+        name=fauxfactory.gen_alphanumeric(8),
+        provision_type=SystemImageType.VM_OR_INSTANCE)
+    collection.delete(False, sys_image_type1, sys_image_type2)
+    view = appliance.browser.create_view(PXESystemImageTypesView)
+    msg = 'System Image Type "{}": Delete successful'.format(sys_image_type1.name)
```

Comment:
**REQUIRED:** These assertions should be happening inside the delete method, not in the test

Reply:
@psav this test verifies flash text. test above verifies deletion. maybe it would be nice to have helping fixture just to create template